### PR TITLE
Create conda recipes for dm-atarchiver and dm-csc-base

### PR DESCRIPTION
**Pull request by srp3rd**

### CODE REVIEW

File: dm-atarchiver/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{% set data= load_setup_py_data() %}
+package:
+  name: dm-atarchiver
+  version: 1.0.0.rc5
+
+source:
+  git_url: https://github.com/lsst-dm/dm_ATArchiver
+  git_rev: master
```
Comment (hwin16) [inline dm-atarchiver/meta.yaml]: This is built using `git master` branch. Wouldn't it be better to use `tagged` tar.gz file? or maybe this file is used for development purposes.

File: dm-csc-base/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{% set data= load_setup_py_data() %}
+package:
+  name: dm-csc-base
+  version: 1.0.0.rc5
+
+source:
+  git_url: https://github.com/lsst-dm/dm_csc_base
+  git_rev: master
```
Comment (hwin16) [inline dm-csc-base/meta.yaml]: This is built using git master branch. Wouldn't it be better to use tagged tar.gz file? or maybe this file is used for development purposes.

**Review by hwin16 — APPROVED** [review]

File: dm-atarchiver/meta.yaml (hunk 1 above)
Comment (srp3rd) [reply dm-atarchiver/meta.yaml]: I switched this to tags/1.0.0-rc6

File: dm-csc-base/meta.yaml (hunk 2 above)
Comment (srp3rd) [reply dm-csc-base/meta.yaml]: I switched this to tags/1.0.0-rc6

**Pull request merged**